### ISSUE TITLE
Broken backward compatibility

### ISSUE THREAD

**Issue by sigorilla** [yannickcr/eslint-plugin-react#1158]
Hi!

For `eslint-plugin-react@6.10.3` you should not use  `jsx-ast-utils@^1.3.4` becase this packages does not support node <4. It contradicts your support. Working version for old node is [1.3.1](https://github.com/evcohen/jsx-ast-utils/commit/efff839e8be73e2028837284de51f9f8b37ce7cf).

Now I get error of eslint:

> Oops! Something went wrong! :(
>
> ESLint couldn't find the plugin "eslint-plugin-react". This can happen for a couple different reasons:
>
> 1. If ESLint is installed globally, then make sure eslint-plugin-react is also installed globally. A globally-installed ESLint cannot find a locally-installed plugin.
>
> 2. If ESLint is installed locally, then it's likely that the plugin isn't installed correctly. Try reinstalling by running the following:
>
>     `npm i eslint-plugin-react@latest --save-dev`
>
> If you still can't figure out the problem, please stop by https://gitter.im/eslint/eslint to chat with the team.

And I get error if run simple script like:

```js
> require('eslint-plugin-react');
TypeError: undefined is not a function
    at Object.<anonymous> (/Users/sigorilla/project/node_modules/jsx-ast-utils/lib/values/expressions/index.js:106:28)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/sigorilla/project/node_modules/jsx-ast-utils/lib/values/index.js:17:20)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)

```

Error in `Object.assign`. With previous version of `jsx-ast-utils` plugin worked correctly.

**Comment by ljharb** [comment]
@sigorilla this is a bug in jsx-ast-utils; see https://github.com/evcohen/jsx-ast-utils/issues/31

**Comment by ljharb** [comment]
Fixed with https://github.com/jsx-eslint/jsx-ast-utils/pull/32
